.gitignore

```diff
@@ -4,3 +4,4 @@
 dist
 NOTES
 MANIFEST
+venv
```

Comment:
If I can object: this is not user specific but project specific, and should be harmless since venv is a general name for a locally used environment.

Not all user projects are python based.

Reply:
I feel like we're splitting hairs here, but so be it, I'll make an update soon.

Reply:
Btw along your reasoning, removing the *.tmproj and _.DS_Store ignores should also apply.

Reply:
Well, I can still modify the PR ;)

I really just want to get my changes released so I can use the offical package instead of my repo.